### PR TITLE
Use Google Maps API key in Frontend Location module.

### PR DESCRIPTION
**Pull request by bartdc**

### CODE REVIEW

File: src/Frontend/Modules/Location/Widgets/Location.php [hunk 1]
```diff
@@ -36,7 +36,14 @@ class Location extends FrontendBaseWidget
...
     {
-        $this->addJS('https://maps.google.com/maps/api/js?sensor=true', true, false);
+        // define Google Maps API key
+        $apikey = $this->get('fork.settings')->get('Core', 'google_maps_key');
+
+        // check Google Maps API key, otherwise show error
+        if ($apikey == null) {
+            trigger_error('Please provide a Google Maps API key.');
+        }
+        $this->addJS('https://maps.googleapis.com/maps/api/js?key=' . $apikey, true, false);
```
Comment (jeroendesloovere) [inline src/Frontend/Modules/Location/Widgets/Location.php]: I've noticed that the sensor=true is gone, does this has implications?

Comment (bartdc) [inline src/Frontend/Modules/Location/Widgets/Location.php]: The sensor parameter is no longer required. Note ate Troubleshooting  https://developers.google.com/maps/documentation/javascript/tutorial#google.maps.Map

**Comment by jeroendesloovere** [comment]
Can't wait until this is merged. Had an error for the first time.
schermafbeelding 2016-06-28 om 17 32 20

**Comment by WouterSioen** [comment]
I've a feeling this changed from a feature to a bug? Since sites break on this now?

**Comment by jeroendesloovere** [comment]
Closes https://github.com/forkcms/forkcms/issues/1430

**Comment by WouterSioen** [comment]
thx

**Pull request merged**